flink-connectors/flink-connector-cassandra/src/test/java/org/apache/flink/streaming/connectors/cassandra/CassandraConnectorITCase.java

```diff
@@ -458,7 +464,10 @@ protected Tuple3<String, Integer, Integer> generateValue(int counter, int checkp
     protected void verifyResultsIdealCircumstances(
             CassandraTupleWriteAheadSink<Tuple3<String, Integer, Integer>> sink) {
 
-        ResultSet result = session.execute(injectTableName(SELECT_DATA_QUERY));
+        ResultSet result =
+                session.execute(
+                        new SimpleStatement(injectTableName(SELECT_DATA_QUERY))
+                                .setReadTimeoutMillis(READ_TIMEOUT_MILLIS));
```

Comment:
Isn't this already set in the cluster builder?

Reply:
Yes but as I wrote in the ticket: 

>  "the driver timeouts were set on the clusterBuilder because previous timeout errors were only on test cases. Here, the request timeouts are on session.execute that do not use the ClusterBuilder. Thus, they use the default 12s timeout. I'll raise to 36s per session request."